### PR TITLE
add a default weight_size

### PR DESCRIPTION
**Pull request by jmitrevs**
# Description

It's possible to read the variable before writing. This sets a default value

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)

## Tests

Standard tests should be enough

## Checklist

- [x] I have read the [guidelines for contributing](https://github.com/fastmachinelearning/hls4ml/blob/main/CONTRIBUTING.md).
- [x] My changes generate no new warnings.
- [x] I have installed and run `pre-commit` on the files I edited or added.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by jmitrevs** [comment]
I believe an error was triggered with batch norms in Quartus, with weight_size being undefined. This fix treats BatchNorm weights as being small, which might not be the best fix in all cases. Nevertheless, we should not read a variable without making sure that each possible branch has it defined.

**Comment by vloncar** [comment]
0.7.0?

**Comment by jmitrevs** [comment]
I think so. It's a 1-line change, which prevents a crash.

**Review by vloncar — APPROVED** [review]
